.github/workflows/build_image_milvus.yml

```diff
@@ -182,6 +181,17 @@ jobs:
           docker push ${{ env.DOCKERHUB_ORG }}/${{ env.DOCKER_IMAGE_NAME }}:${{ steps.latest_tag.outputs.tag_ref }}
           docker tag ${{env.DOCKERHUB_ORG}}/${{ env.DOCKER_IMAGE_NAME }}:${{ steps.latest_tag.outputs.tag_ref }} 219776070323.dkr.ecr.ap-south-1.amazonaws.com/${{ env.DOCKER_IMAGE_NAME }}:${{ steps.latest_tag.outputs.tag_ref }}
           docker push 219776070323.dkr.ecr.ap-south-1.amazonaws.com/${{ env.DOCKER_IMAGE_NAME }}:${{ steps.latest_tag.outputs.tag_ref }}
+          
+          while IFS=$'\t' read -r image function file _; do
+            if [[ "$function" == "dockerBuild" && ! -z "$file" ]]; then
+              echo "Building $image Image"
+              docker build -f $file --no-cache -t ${{env.DOCKERHUB_ORG}}/$image:${{ steps.latest_tag.outputs.tag_ref }} .
+              docker push ${{env.DOCKERHUB_ORG}}/$image:${{ steps.latest_tag.outputs.tag_ref }}
```

Comment:
The tag and push is duplicated in line 195 again. I think all you need to do is build the image. then the second loop will take care of tagging and pushing. 

Also since its an image created by us, it need to be scanned for vulnerabilities.

Reply:
Second loop loops over consumes there's no mention of bookie and zookeeper, that's why and in first loop only going over artifacts which have dockerfile present.